httpreq/httpreq.go

```diff
@@ -76,6 +96,18 @@ func (r *HttpReq) WithToken(token string) *HttpReq {
 	return r
 }
 
+func (r *HttpReq) CallWithTransport(transport *http.Transport) *HttpReq {
```

Comment:
Do not change `defaultClient`.
Just run `http.Client.Do()` using `transport` parameter.

```
func (r *HttpReq) CallWithTransport(transport *http.Transport, v interface{}) (int, error) {
   // create new *http.Client using transport parameter.
   // and run Client.Do
}
```

Reply:
I modify CallWithTransport method
and added CallWithClient method

Reply:
No. Do not change `defaultClient`.
Only `CallWithTransport` use `transport` parameter.
Others should use default Transport.

If we need change `defaultClient.Transport`, add `SetGlobalTransport` function.

Reply:
I added the method CallWithTransport and SetGlobalTransport